### ISSUE TITLE
Difficulty

### ISSUE THREAD

**Issue by Tsilliev** [RaZeR-RBI/KAG-ZombieFP#1]
Hello, I have been playing KAG on and off since around 2012, came back recently to see my favorite game mode Zombie Fortress.
I see there are a lot of new things which is great! 
But is there a chance to tweak the difficulty? By night 3 or night 4 we are overrun by zombies, especially by the spinning zombie that breaks everything so fast.
An idea would be to spawn zomie cats, chickens and other small zombies for the first 7 days, at the 7th day release a boss or 1 flying spinning zombie, then spawn higher tier zombies, etc.
Give chance for players to build up the base in the beginning and maybe reduce the dmg to blocks that the flying spinning zombie does?

**Comment by RaZeR-RBI** [comment]
Sorry for the late reply.

I've made some changes to the digger (the flying spinning zombie) recently, along with a ton of fixes and balance changes, I recommend you to check them out :wink:

By the way, most people were winning just fine before the changes as long as they were placing/moving the base properly and were aware of their surroundings - I've successfully played it with teams consisting from 3 to 8 people. I guess some newbie players just think that this mod is easy or that the more experienced players would win it for them and just go building castles, towers and stuff. Teamwork is the key :wink:

The reason why diggers do that much damage is that original mod was too easy to 'cheese' through - you basically make an underground camp and zombies wouldn't reach you, which makes people play like it's Minecraft for some reason I just don't understand.

I've switched the pacing to make it more session-like, something in spirit of Left 4 Dead and similar PvE session-based games - you have a map, and you need to survive and adapt to the surroundings to complete the challenge - which, in my opinion, makes the mod more dynamic and gives you a reason to play it.

TL;DR - Difficulty should be lower now, so the new players can play it too.
If you think that this issue still not resolved, feel free to open it again. Cheers!
